packages/rum-core/src/rumEvent.types.ts

```diff
@@ -276,6 +282,96 @@ export type RumErrorEvent = CommonProperties &
         }
         [k: string]: unknown
       }
+      /**
+       * Description of each thread in the process when error happened.
+       */
+      threads?: {
```

Comment:
❔ **Question**: wouldn't it be straightforward to type a single thread first, then reuse that type for the `threads` attribute ?

Reply:
Those types are generated automatically from the JSON schema.
Not sure if we can influence the generator to produce types with a certain style.
Do you see any issue with it?

Reply:
> Do you see any issue with it?

Not really, I was just thinking if we ever wanted to use the type of `thread`, it would be straightforward to have a dedicated one instead of extracting it from the parent type.

Reply:
>  if we ever wanted to use the type of thread

On the browser side, we probably won't 🙂

Actually, I think a thread schema could be extracted to generate a thread type if we need it at some point.